pipelines/diaTractQualityCore.yaml

```diff
@@ -0,0 +1,17 @@
+description: |
+  Tier 1 metrics to assess difference image analysis (DIA) quality.
+  This is designed to analyze the `diaSourceTable_tract` dataset, which
+  is a consolidated version of `{fakesType}{coaddName}Diff_assocDiaSrcTable`,
+  which is output from DrpAssociationPipeTask on a per-patch-and-tract basis.
+
+tasks:
+  analyzeDiaSourceTableTract:
+    class: lsst.analysis.tools.tasks.DiaSourceTableTractAnalysisTask
+    # This task is designed to automatically run all the relevant metric
+    # atools, so only plots need to be configured here.
+    config:
```

Comment:
I think it would be better if the metrics were not run in the task. That way if other people want to use the task to read in this sort of data the metrics don't get run.